src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -145,6 +155,7 @@ def __call__(self, poller):
 
         logger.info("Long running operation '%s' completed with result %s",
                     self.start_msg, result)
+        self.controller.end()
```

Comment:
Will any exception cause the progress controller not be ended? What's the consequence?